app/assets/javascripts/controllers/credentials/credentials_controller.js

```diff
@@ -4,7 +4,7 @@ ManageIQ.angular.app.controller('CredentialsController', ['$scope', '$attrs', fu
   vm.vmScope = function() {
     return $scope.$eval($attrs.vmScope);
   };
-  
+
```

Comment:
Oh, sorry, there was a trailing whitespace.. I guess we don't need to remove it.. will do :)